ChangeLog

```diff
@@ -1,12 +1,21 @@
-Changes in squid-6.4 (15 Oct 2023):
+Changes in squid-6.4 (22 Oct 2023):
 
 	- Regression: Restore support for legacy cache_object cache manager requests
 	- Regression: Do not use static initialization to register modules
 	- Bug 5301: cachemgr.cgi not showing new manager interface URLs
 	- Bug 5300: cachemgr.cgi assertion
+	- Fix stack buffer overflow when parsing Digest Authorization
+	- Fix userinfo percent-encoding
+	- Fix store_client caller memory leak on certain errors
+	- Fix validation of certificates with CN=*
+	- Remove serialized HTTP headers from storeClientCopy()
+	- Replaced clientReplyContext::tempBuffer with old_reqofs
```

Comment:
Thank you for addressing my blocking concern. I am not against this PR merging with the current ChangeLog entries, but I recommend replacing these two developer-focused items with a single one targeting Squid admins (i.e. the primary ChangeLog audience):

```suggestion
	- Fix handling of large stored response headers
```

The suggested replacement matches the corresponding [vulnerability](https://megamansec.github.io/Squid-Security-Audit/cache-headers.html) title.

Again, I do not insist on this replacement.

P.S. I trust you are aware that this PR requires a merge conflict resolution, rendering any before-resolution approvals ineffective.